### PR TITLE
Fix blood cultists being able to place structures on survival pods

### PR DESCRIPTION
**Pull request by XeonMations**
## About The Pull Request

Fixes an oversight in blood cultist metal structure creation.

## Why It's Good For The Game

Fixes an oversight that blood cultists could previously place structures on survivalpods due to them being considered on station premises and a valid blob area.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

https://github.com/BeeStation/BeeStation-Hornet/assets/62395746/2c5c6545-603f-4f60-a3ca-01adf3a07831

</details>

## Changelog
:cl:
fix: Fixed blood cultists being able to place structures on survival pods
/:cl:

### CODE REVIEW

**Comment by St0rmC4st3r** [comment]
This was not intended????

File: code/game/objects/items/stacks/sheets/mineral/metals.dm [hunk 1]
```diff
@@ -94,7 +94,7 @@ Metals Sheets
 		return
 	var/turf/T = get_turf(user) //we may have moved. adjust as needed...
 	var/area/A = get_area(user)
-	if((!is_station_level(T.z) && !is_mining_level(T.z)) || (A && !(A.area_flags & BLOBS_ALLOWED)))
+	if((!is_station_level(T.z) && !is_mining_level(T.z)) || (A && !(A.area_flags & BLOBS_ALLOWED)) || (A && !(A.area_flags & VALID_TERRITORY)))
```
Comment (EvilDragonfiend) [inline code/game/objects/items/stacks/sheets/mineral/metals.dm]:
```suggestion
	if((!is_station_level(T.z) && !is_mining_level(T.z)) || (A && !(A.area_flags & (BLOBS_ALLOWED | VALID_TERRITORY))))
```

**Review by EvilDragonfiend — CHANGES_REQUESTED** [review]

**Comment by EvilDragonfiend** [comment]
I am not sure which thing people generally like.

**Review by EvilDragonfiend — APPROVED** [review]
Code is fine but I am not sure the change. Maybe cultists should stop being asshole with this?? Although it closes possibilities.

**Pull request merged**